apps/web/public/restricted.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="UTF-8" />
```

Comment:
See: https://validator.w3.org/nu/#textarea

Trailing slash on void elements [has no effect](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-in-void-element-start-tags-do-not-mark-the-start-tags-as-self-closing) and [interacts badly with unquoted attribute values](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-directly-preceded-by-unquoted-attribute-values).

```suggestion
    <meta charset="UTF-8">
```

Reply:
Yeah, I saw that, too.   It's actually an info message, not an error.  VS Code automatically adds that on save so it will be automatically changed back every time that page is interacted with unless we want to change linting rules.  According to the validator help link "my formatting tools adds it" and "I work with JSX" are valid reasons for adding the slash.  Along with, apparently, "I like how it looks"?  :laughing: